test/jdk/java/lang/management/MemoryMXBean/LowMemoryTest2.java

```diff
@@ -31,6 +31,19 @@
  * is generated.
  */
 
+/*
+ * @test
+ * @bug     4982128
+ * @summary Test low memory detection of non-heap memory pool
+ *
+ * @run build MemoryUtil
+ *
+ * @run main/othervm/timeout=600 -noclassgc -XX:MaxMetaspaceSize=32m
```

Comment:
I would prefer to have multiply tests rather than multiply 'run' in single tests. It allow jtreg to fail only one test which fail and run them parallel.